day_12/task_1/hot_springs.cpp

```diff
@@ -1 +1,47 @@
 #include "hot_springs.hpp"
+#include <sstream>
+#include <algorithm>
+
+std::vector<int> extract_groups(std::string spring_condition_data)
+{
+    std::replace(spring_condition_data.begin(), spring_condition_data.end(),',', ' ');
+    std::stringstream ss{spring_condition_data};
+    std::string conditions;
+    std::vector<int> groups(3);
+    ss >> conditions >> groups[0] >> groups[1] >> groups[2];
```

Comment:
what happened when will be 4 groups?

Reply:
Then it would be hard to modify. So I changed it into for loop.